src/uu/df/src/blocks.rs

```diff
@@ -220,56 +208,64 @@ mod tests {
 
     use std::env;
 
-    use crate::blocks::{to_magnitude_and_suffix, BlockSize};
+    use crate::blocks::{to_magnitude_and_suffix, BlockSize, SuffixType};
 
     #[test]
     fn test_to_magnitude_and_suffix_powers_of_1024() {
```

Comment:
I have to admit, it's pretty difficult to follow the logic of `HumanReadable`, `SizeFormat`, `BlockSize`, `SuffixType`, and more, but if I'm reading this pull request correctly, we are just decoupling the value of the number (1024, 2048, 4096, etc.) from the decision of what suffix to apply (`SuffixType::IEC`, `SuffixType:SI`, etc.). So we could now call `to_magnitude_and_suffix(1024, SuffixType::SI)` if we wanted to. Is that right?

Reply:
Yes, that's correct.

Your comment made me rethink the enums you mentioned and I realized that `SizeFormat` is not necessary and I turned it into an `Option<HumanReadable>` with the latest push. I hope this reduces some of the complexity.